src/SaveDialog.vala

```diff
@@ -249,4 +249,11 @@ public class Screenshot.SaveDialog : Granite.Dialog {
             folder_image.gicon = new ThemedIcon ("folder");
         }
     }
+
+    public void hide_destroy () {
+        hide ();
+        GLib.Timeout.add_once (500, () => {
```

Comment:
I don't really love magic number timeouts. Is there something more reliable we can tie this to or otherwise comment why this is here and why it's 500?

Reply:
It seems that the advantage closes before the process of copying to the clipboard is finished, there is no response indicator that the process was executed correctly, let me look for something more efficient.